tool.html

```diff
@@ -13,7 +13,7 @@
 		<!--TODO: Separate each of the below tempates into its own directive/components following the principles
 			of modularity and separation of concerns.-->
 		<div class='heading-message align-text-center main-title'>Microbiome Diversity Inspector</div>
-		<div class='heading-message align-text-center title-description'>A platform for visual analysis of microbiome data</div>
+		<div class='heading-message align-text-center title-description'>A tool for visual analysis of microbiome data</div>

```

Comment:
Revert this change (you probably didn't sync your local copy with this commit - https://github.com/gbelwariar/Microbiome-Diversity-Inspector/commit/059459770f35414dd2db0a702c50c2ba3ff76343), i.e - change - `tool` to `platform`. I should have notified you about this. My mistake :(